react/features/conference/components/native/carmode/styles.js

```diff
@@ -0,0 +1,208 @@
+import BaseTheme from '../../../../base/ui/components/BaseTheme.native';
+
+/**
+ * The default size of the microphone icon.
+ */
+const DEFAULT_MICROPHONE_SIZE = 180;
+
+const TITLE_BAR_BUTTON_SIZE = 24;
+
+/**
+ * Base button style.
+ */
+const baseButton = {
+    borderRadius: BaseTheme.shape.borderRadius,
+    height: BaseTheme.spacing[7],
+    marginTop: BaseTheme.spacing[3],
+    marginLeft: BaseTheme.spacing[10],
+    marginRight: BaseTheme.spacing[10],
+    display: 'flex',
+    justifyContent: 'space-around',
+    width: 300
+};
+
+/**
+ * Base label style.
+ */
+const baseLabel = {
+    display: 'flex',
+    fontSize: 16,
+    textTransform: 'capitalize'
+};
+
+/**
+ * The styles of the safe area view that contains the title bar.
+ */
+const titleBarSafeView = {
+    left: 0,
+    position: 'absolute',
+    right: 0,
+    top: 0
+};
+
+/**
+ * The styles of the native components of Carmode.
+ */
+export default {
+
+    bottomContainer: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        bottom: 0,
+        left: 0,
+        right: 0,
+        position: 'absolute'
+    },
+
+    /**
+     * {@code Conference} Style.
+     */
+    conference: {
+        alignSelf: 'stretch',
+        backgroundColor: BaseTheme.palette.uiBackground,
+        flex: 1,
+        justifyContent: 'center',
+        alignItems: 'center'
+    },
+
+    microphoneStyles: {
+        container: (size: number = DEFAULT_MICROPHONE_SIZE) => {
+            return {
+                borderRadius: size / 2,
+                height: size,
+                maxHeight: size,
+                justifyContent: 'center',
+                overflow: 'hidden',
+                width: size,
+                maxWidth: size,
+                flex: 1,
+                zIndex: 1,
+                elevation: 1
+            };
+        },
+
+        icon: (size: number = DEFAULT_MICROPHONE_SIZE) => {
```

Comment:
Do we set multiple values here?

Reply:
We only use it in one place. Shall I just hardode?

Reply:
Yes please.

Reply:
done